descwl_coadd/coadd.py

```diff
@@ -89,6 +94,136 @@ def make_coadd_obs(
 def make_coadd(
     exps, coadd_wcs, coadd_bbox, psf_dims, rng, remove_poisson,
     max_maskfrac=MAX_MASKFRAC,
+    is_warps=False,
```

Comment:
I am not a fan of the `is_warps` option. The coaddition is going to look very differently if it reads in the DM `warp` or `calexp` that I don't think a single function should try to do both.

Reply:
we don't want to break existing codes

Reply:
And we don't want the same code copied in multiple places, which means maintaining the same code in two places